guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -2,8 +2,8 @@
 = Installing PostgreSQL
 
 You can install only the same version of PostgreSQL that is installed with the `{foreman-installer}` tool during an internal database installation.
-You can install PostgreSQL using {RHELServer} 7 repositories or from an external source, as long as the version is supported.
-{Project} supports PostgreSQL version 12.1.
+You can install PostgreSQL using {EL} 8 repositories or from an external source, as long as the version is supported.
```

Comment:
Given we have `postgresql-evr`, is an external source really a valid option?
```suggestion
You can install PostgreSQL using {EL} 8 repositories.
```

Though given we previously already gave instructions to set up repositories, perhaps it's better to drop it.
```suggestion
```